benches/generate_and_query/random_points.rs

```diff
@@ -12,24 +12,26 @@ pub struct RandomPointsOnEarth {
     rng: StdRng,
     half_width: f64,
     half_height: f64,
-    ecef_from_local: Matrix4<f64>,
+    ecef_from_local: Isometry3<f64>,
+    ecef_from_local_mat: Matrix4<f64>,
     size: usize,
     count: usize,
 }
 
-pub const SEED: u64 = 80_293_751_232;
-
 impl RandomPointsOnEarth {
     pub fn new(width: f64, height: f64, size: usize, seed: u64) -> Self {
         let mut rng = StdRng::seed_from_u64(seed);
         let lat = rng.gen_range(-90.0, 90.0);
         let lon = rng.gen_range(-180.0, 180.0);
         let ecef_from_local = local_frame_from_lat_lng(lat, lon);
```

Comment:
I just looked at it again. Actually the fix is here, we need
```rust
let ecef_from_local = local_frame_from_lat_lng(lat, lon).inverse();
```
Then you can change back to `rect_bound()` below, as it's not a bug in s2.

Reply:
Hmm, why did changing it to cap_bound fix it then?

Reply:
I didn't investigate this further, but what I found ot is:
* Comparison of the behavior of the c++ and the rust version of `rect_bound`, which was exactly the same for the failing values.
* I looked at the `ecef_from_local` translations, which only made sense for the inverse.

Reply:
So `local_frame_from_lat_lng()` actually doesn't return `ecef_from_local` but `local_from_ecef`?

Reply:
yes, see also https://github.com/lyft/avmapping-geometry/commit/395c119a92af7b3ad4a24d2760092e8124146e88#diff-fcd43e43678aa473c40a0a7be1f9f558R90